src/main/java/org/folio/circulation/domain/policy/LostItemPolicy.java

```diff
@@ -25,6 +25,10 @@ public static LostItemPolicy unknown(String id) {
     return new UnknownLostItemPolicy(id);
   }
 
+  public boolean isUnknown() {
```

Comment:
Why is this needed, as opposed to using polymorphism?

Reply:
This method is no longer relevant, so I removed completely.